cli/cook/subcommands/submit.py

```diff
@@ -210,6 +210,8 @@ def register(add_parser, add_defaults):
     submit_parser.add_argument('--ports', help='number of ports to reserve for job', type=int)
     submit_parser.add_argument('--application-name', '-a', help='name of application submitting the job')
     submit_parser.add_argument('--application-version', '-v', help='version of application submitting the job')
+    submit_parser.add_argument('--executor', '-E', help='executor to use to run the job on the Mesos agent',
+                               choices=('cook', 'mesos'))
```

Comment:
Should we also add the `progress-output-file` and `progress-regex-string` flags? I'm okay if you feel it needs to be a separate PR.

Reply:
Created issue #508 for this